manifests/kustomize/base/installs/generic/pipeline-install-config.yaml

```diff
@@ -11,7 +11,7 @@ data:
     until the changes take effect. A quick way to restart all deployments in a
     namespace: `kubectl rollout restart deployment -n <your-namespace>`.
   appName: pipeline
-  appVersion: 2.0.0
+  appVersion: 2.0.1
```

Comment:
It seems that we should still use `2.0.0` for the consistency across repo.

Reply:
This was changed due to version 2.0.1 was released yesterday, see https://github.com/kubeflow/pipelines/pull/9899.